AllReadyApp/Web-App/AllReady/Areas/Admin/Controllers/SiteAdminController.cs

```diff
@@ -61,7 +61,9 @@ public IActionResult Index()
         [ValidateAntiForgeryToken]
         public async Task<IActionResult> ConfirmDeleteUser(string userId)
         {
-            await _mediator.SendAsync(new DeleteUserCommand { UserId = userId });
+            var user = await _userManager.FindByIdAsync(userId);
+            await _userManager.DeleteAsync(user);

```

Comment:
Hrm...well, we don't want a handler for the sake of a handler, but all that extra cruft has to be taken care of somehow. And our only other option here would be to revert to injecting the DBContext (or repository) into the controller. 

As far as the delete goes, the only occurances 
 - are in Admin 
 - might be in account self-termination (down the road)
 - are likely edge cases

So, IMO I'm okay leaving the delete here for now and we can likely start to raise the event when we get to the point where we're taking care of the extraneous data. For now, there's no keys/integrity defined afaik when we nuke a user, it  just leaves those rows orphaned. Again, since it's an edge case and currently only in admin, I think we're okay.

Sorry for over-thinking it ;)

:shipit: